src/containers/project-saver.jsx

```diff
@@ -33,11 +33,7 @@ class ProjectSaver extends React.Component {
         document.body.appendChild(saveLink);
 
         this.props.saveProjectSb3().then(content => {
-            // TODO user-friendly project name
-            // File name: project-DATE-TIME
-            const date = new Date();
-            const timestamp = `${date.toLocaleDateString()}-${date.toLocaleTimeString()}`;
-            const filename = `untitled-project-${timestamp}.sb3`;
+            const filename = `${this.props.projectTitle}.sb3`;
```

Comment:
The behavior is kind of weird if the title field is blank (downloads sb3.zip)

Reply:
Hm, good point. I'll give it a default in that case.